bottlecap/src/config/mod.rs

```diff
@@ -108,6 +106,13 @@ pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
         _ => ConfigError::ParseError(err.to_string()),
     })?;
 
+    // TODO(duncanista): revert to using datadog.yaml when we have a proper serializer
+    if path.exists() {
+        log_failover_reason("datadog_yaml");
+        debug!("datadog.yaml is not supported, use environment variables instead");
+        return Err(ConfigError::UnsupportedField("datadog_yaml".to_string()));
```

Comment:
. instead of _?

Reply:
This is going to be sent as a metric, so the tag of the metric I think cannot contain `.`

Reply:
oh ok sounds good